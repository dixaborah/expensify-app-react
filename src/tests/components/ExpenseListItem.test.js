import React from 'react'
import { shallow } from 'enzyme'

import ExpenseListItem from '../../views/ExpenseListItem'
import Expenses from '../fixtures/Expenses'



test('Should render ExpenseListItem correctly', () => {
    const Expense = Expenses[1]
    const wrapper = shallow(<ExpenseListItem { ...Expense } />)
    expect(wrapper).toMatchSnapshot()
})