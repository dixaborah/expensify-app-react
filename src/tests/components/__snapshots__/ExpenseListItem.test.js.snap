// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $
    1095
     - 
    Dec 31, 2016
  </p>
</div>
`;
